lib/api/handlers/pokemon_handler.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:convert';
+
+import 'package:counter_async_redux/api/models/detailsPokemon_model.dart';
+import 'package:counter_async_redux/api/models/pokemon_model.dart';
+import 'package:counter_async_redux/utilities/constants.dart';
+import 'package:http/http.dart' as http;
+
+class PokemonHandler {
+  static Future<List<Pokemon>?> getPokemons(String? nextPage) async {
+    var response = http.Response('', 100);
+    if (nextPage == null) {
```

Comment:
this whole if statement can be simplified. Instead of checking if nextPage is null here, forgo with the if else statement, and check if nextPage is null inside the `Uri.tryParse()`